extemporaneous_grading/conf/locale/es_419/LC_MESSAGES/text.po

```diff
@@ -17,14 +17,122 @@ msgstr ""
+"learner will see the content without any restrictions."
+msgstr ""
+"La fecha de entrega para este componente. Si el estudiante accede antes de "
+"esta fecha, verá el contenido sin restricciones."
+
+#: extemporaneous_grading.py:58
+msgid "Due Time (UTC)"
+msgstr "Hora de Entrega (UTC)"
+
+#: extemporaneous_grading.py:59
+msgid "The time of the due date. The valid format is HH:MM."
+msgstr "La hora de la fecha de entrega. El formato válido es HH:MM."
+
+#: extemporaneous_grading.py:65
+msgid "Late Due Date"
+msgstr "Fecha de Entrega Tardía"
+
+#: extemporaneous_grading.py:67
+msgid ""
+"The late due date for this component. If the learner access after the due "
+"date and before the late due date, the learner will see a message that the "
+"submission is late. If the learner accepts the late submission pressing the "
+"button, the learner will see the content."
+msgstr ""
+"La fecha de entrega tardía para este componente. Si el estudiante accede "
+"después de la fecha de entrega y antes de la fecha de entrega tardía, verá "
+"un mensaje que la entrega es tardía. Si el estudiante acepta la entrega "
+"tardía presionando el botón, verá el contenido."
+
+#: extemporaneous_grading.py:77
+msgid "Late Due Time (UTC)"
+msgstr "Hora de Entrega Tardía (UTC)"
+
+#: extemporaneous_grading.py:78
+msgid "The time of the late due date. The valid format is HH:MM."
+msgstr "La hora de la fecha de entrega tardía. El formato válido es HH:MM."
+
+#: extemporaneous_grading.py:84
+msgid "Due Date Explanation Text"
+msgstr "Texto Explicativo de la Fecha de Entrega"
+
+#: extemporaneous_grading.py:85
+msgid ""
+"The explanation text that will be shown to the learner when the due date is "
+"passed."
+msgstr ""
+"El texto explicativo que se mostrará al estudiante cuando la fecha de "
+"entrega haya pasado."
+
+#: extemporaneous_grading.py:88
+msgid ""
+"The due date has passed. You can still submit this assignment until the late "
+"due date. After the late due date, you will not be able to submit this "
+"assignment. If you accept the late submission, press the button."
+msgstr ""
+"La fecha de entrega ha pasado. Aún puedes entregar esta tarea hasta la fecha "
+"de entrega tardía. Después de la fecha de entrega tardía, no podrás entregar "
+"esta tarea. Si aceptas la entrega tardía, presiona el botón."
+
+#: extemporaneous_grading.py:95
+msgid "Late Due Date Explanation Text"
+msgstr "Texto Explicativo de la Fecha de Entrega Tardía"
+
+#: extemporaneous_grading.py:96
+msgid ""
+"The explanation text that will be shown to the learner when the late due "
+"date is passed."
+msgstr ""
+"El texto explicativo que se mostrará al estudiante cuando la fecha de "
+"entrega tardía haya pasado."
+
+#: extemporaneous_grading.py:98
+msgid ""
+"The late due date has passed. You can not submit this assignment anymore."
+msgstr ""
+"La fecha de entrega tardía ha pasado. Ya no puedes entregar esta tarea."
+
+#: extemporaneous_grading.py:102
+msgid "Is Late Submission"
```

Comment:
The grammatical correct way is:
Is it a late submission?
So I think we can use: Late Submission instead

Reply:
Done: https://github.com/eduNEXT/xblock-extemporaneous-grading/commit/aaed9daa91de0eebf250d0cba14446801701ab82